### PR TITLE
feat: Single message copy as children props

### LINKED ISSUE #29: Review SingleMessage
Add extra fields and variations.

Make it render children instead hard coded fields.

### PR DESCRIPTION
**Pull request by gusliedke**
BREAKING CHANGE: Single message uses children as props

Type: feature

Fixes #29

### Variations
- [x] Copy First (Copy left) default Copy right
- [x] Copy Only (no Image)
- [x] Full Width (Full width image) copy first or copy "second"

Screenshot 2019-07-08 at 11 55 19

### CODE REVIEW

File: src/components/Atoms/Picture/Picture.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ const Picture = ({ images, alt, theme, width, height, objectFit }) => {
 };
 
 Picture.propTypes = {
-  images: PropTypes.object.isRequired,
+  images: PropTypes.oneOfType([PropTypes.string, PropTypes.object]).isRequired,
```
Comment (AndyEPhipps) [inline src/components/Atoms/Picture/Picture.js]: I thought `object` wasn't allowed in the ESlint rules we're using?

Comment (gusliedke) [reply src/components/Atoms/Picture/Picture.js]: It's not .. there is a ```eslint-disable``` line at the top :) .. let me fix that with shape

Comment (gusliedke) [reply src/components/Atoms/Picture/Picture.js]: ok, it was ignoring due `theme` type .. I've added a shape type for `theme` as well . I think it works.

**Pull request merged**